thirdeye/thirdeye-frontend/app/utils/build-tooltip.js

```diff
@@ -66,12 +66,12 @@ const getValues = (hoverUrns, hoverTimestamp, lookup, entities) => {
   metricUrns.forEach(urn => {
     // find first smaller or equal element
     const currentLookup = (lookup[toCurrentUrn(urn)] || []).reverse();
-    const currentTimeseries = currentLookup.find(t => t[0] <= hoverTimestamp && t[1] != null);
-    const current = currentTimeseries ? currentTimeseries[1] : parseFloat('NaN');
+    const currentTimeseries = currentLookup.find(t => t[0] <= hoverTimestamp);
+    const current = currentTimeseries ? currentTimeseries[1] ? currentTimeseries[1] : Number.NaN : Number.NaN;
```

Comment:
NIP: for clarity
 const current = currentTimeseries ? currentTimeseries[1] ? currentTimeseries[1] : Number.NaN : Number.NaN;
to
 const current = currentTimeseries ? ( currentTimeseries[1] ? currentTimeseries[1] : Number.NaN ) : Number.NaN;
or this i think should work
 const current = currentTimeseries && currentTimeseries[1] ? currentTimeseries[1] ? : Number.NaN;

Reply:
fixed